src/main/java/com/gb/cornucopia/proxy/ClientProxy.java

```diff
@@ -17,7 +17,6 @@
 	@Override
 	public void preInit(final FMLPreInitializationEvent e) {
 		super.preInit(e);
-		InvModel.register();
```

Comment:
are you doing registry differently now?

Reply:
Yep! The way it's handled in newer versions of Forge is you use `@EventBusSubscriber` to register the `RegistrationHandler` class, [which you can see here](https://github.com/team-soas/cornucopia/pull/80/files#diff-788dd1c7503c783b95b1d13041489f91R58).